server/main_server/src/rodsServer.cpp

```diff
@@ -2250,7 +2287,7 @@ void task_spawn_manager()
         }
 
         boost::unique_lock<boost::mutex> spwn_req_lock(SpawnReqCondMutex);
-        SpawnReqCond.wait(spwn_req_lock);
+        SpawnReqCond.wait(spwn_req_lock); // TODO This form of wait is not recommended due to spurious wake ups, etc.
```

Comment:
While not the best, iRODS has used this form of `wait` for quite some time. Therefore, we'll leave this as is.

Reply:
Create an issue for this form of `wait`.

Reply:
Done.